pkg/virt-controller/watch/migration_test.go

```diff
@@ -1424,6 +1424,28 @@ var _ = Describe("Migration watcher", func() {
 				Expect(matchedPolicy).To(BeNil())
 			})
 
+			It("label with an empty value should match any value", func() {
+				const labelKey = "mp-key-0"
+				labelValue := rand.String(5)
+
+				By("Defining a policy that matches a key with empty value")
+				policy := kubecli.NewMinimalMigrationPolicy("testpolicy")
+				policy.Spec.Selectors = &migrationsv1.Selectors{
+					VirtualMachineInstanceSelector: &metav1.LabelSelector{
```

Comment:
Can we check the namespace labels too? WDYT? Thanks

Reply:
Good idea, done.
Thanks for your review!